Original issue only affect tests. Probability that somebody uses shelve/pickle 
out in the wild is fairly low.

--- a/setup.cfg
+++ b/setup.cfg
@@ -46,7 +46,7 @@ install_requires =
 	packaging>=20.9
 	pyyaml>=4.2b1
 	setuptools>=19.6
-	sympy!=1.9,>=1.2  # see https://github.com/sympy/sympy/issues/22241
+	sympy>=1.2
 	toml>=0.10.2
 	tqdm>=3.4.0
 	unyt>=2.8.0
